package.json

```diff
@@ -45,7 +45,9 @@
     "fs-extra": "10.1.0",
     "logsnag": "^0.1.5",
     "mime": "^3.0.0",
-    "prettyjson": "^1.2.5"
+    "prettyjson": "^1.2.5",
+    "prompt-sync": "^4.2.0",
```

Comment:
added prompt-sync to ask for confirmation before cleanup. And semver to have easier range comparison in the cleanup method to list which versions to remove.